[debug] [2025-09-02T10:32:51.704Z] ----------------------------------------------------------------------
[debug] [2025-09-02T10:32:51.707Z] Command:       C:\nvm4w\nodejs\node.exe C:\Users\alsar\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2025-09-02T10:32:51.708Z] CLI Version:   14.15.1
[debug] [2025-09-02T10:32:51.708Z] Platform:      win32
[debug] [2025-09-02T10:32:51.708Z] Node Version:  v20.18.0
[debug] [2025-09-02T10:32:51.708Z] Time:          Tue Sep 02 2025 13:32:51 GMT+0300 (GMT+03:00)
[debug] [2025-09-02T10:32:51.709Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-09-02T10:32:51.715Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-02T10:32:51.715Z] > authorizing via signed-in user (movo.ems@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  C:\projects\MOVO\back\Codes\backend

Before we get started, keep in mind:

  * You are currently outside your home directory

[debug] [2025-09-02T10:33:03.893Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (C:\Users\alsar\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\@inquirer\core\dist\commonjs\lib\create-prompt.js:101:37)
    at Interface.emit (node:events:519:28)
    at Interface.emit (node:domain:488:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1126:18)
    at ReadStream.onkeypress (node:internal/readline/interface:265:20)
    at ReadStream.emit (node:events:531:35)
    at ReadStream.emit (node:domain:488:12)
    at emitKeys (node:internal/readline/utils:371:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
